solutions/fully-configurable/outputs.tf

```diff
@@ -7,6 +7,11 @@ output "event_notification_instance_name" {
   value       = var.existing_event_notifications_instance_crn == null ? module.event_notifications[0].event_notification_instance_name : data.ibm_resource_instance.existing_en_instance[0].name
 }
 
+output "en_instance_crn" {
+  description = "The CRN of event notifications instance"
+  value       = local.use_existing_en_instance ? [{ en_crn = var.existing_event_notifications_instance_crn }] : [{ en_crn = module.event_notifications[0].crn }]
```

Comment:
As per this [comment](https://github.com/terraform-ibm-modules/terraform-ibm-cloud-logs/pull/64/files#r2133865041), I am proposing we call the option `crn` instead of `en_crn`. Can you update this?